libr/hash/luhn.h

```diff
@@ -0,0 +1,6 @@
+#ifndef _LUHN_H_
+#define _LUHN_H_
+
+R_API ut64 r_hash_luhn(const ut8 *buf, ut64 len);
```

Comment:
put this in libr/include/r_hash.h 

this is the wrong place to put include files